gandalf.js

```diff
@@ -1,7 +1,8 @@
-/**
```

Comment:
SCRIPT-Well done with the efficient error handling, it covers all the possible scenarios.